components/modules/node.c

```diff
@@ -245,6 +245,46 @@ static int node_output(lua_State *L) {
     return 0;
 }
 
+// The implementation of node.osoutput redirect all OS logging to LUA space
```

Comment:
```suggestion
// The implementation of node.osoutput redirect all OS logging to Lua space
```
Lua is not an acronym 😉